### DISCUSSION TITLE
特定の列のキーが反応しない

### DISCUSSION THREAD

**Discussion by tom-ok1** [Yowkees/keyball#569] [Q&A]
お世話になっております
keyball 61 rev2 を購入し組み立てたのですが数字の４があるキーの列（r,f,v）が特定の条件下で反応しません
下記を確認済みです
- 最初に基盤にType-cを指して4,r,f,vを押すと反応しない
- 1度基盤側に行き、4,r,f,vのいずれかのキーの鉄の部分を指で優しく触れると全てのキーが反応するようになる
- 時間が経つと突然全てのキーが反応しなくなるor反応が悪くなる

左右の基盤を入れ替えてみたのですが改善しませんでした。。（この時ボールサイドは正常に動いています）
基盤に触れるだけで動作が改善するので半田付け以外に問題があるのかもと思っていますが検討がつかず質問させていただきました。

お手数ですが何か似た事例や改善すべき点などがわかればご教示いただけませんでしょうか...
よろしくお願いいたします。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。

Keyball61右手側の縦一列（4,r,f,v）でのキーの不具合について了解しました。
一度はんだ付けを確認させていただきたいので、お手数おかけしますが基板裏側の写真の添付をお願いしてもよろしいでしょうか。

**Reply by Ryo624** [reply]
ひとまず確認できることとしては、添付させていただいた写真の赤丸の部分が不具合が起きているキー一列とつながっているダイオードになっていますので、こちらのはんだ付けをやり直しを行い、改善が見られなければ取り替えてみてください。
また、基板裏側のProMicro用の12ピンソケットをはんだ付けした箇所にCOL１と書かれた端子があると思いますので、そちらも端子にしっかりはんだが馴染んでいるか、隣の端子にはんだが触れていないかの確認をお願いいたします。
![kb61](https://github.com/Yowkees/keyball/assets/143698349/f0ffbe01-9ccc-4494-80e3-7009039ea0a8)

**Reply by tom-ok1** [reply]
お早いご返事ありがとうございます
添付の画像の赤丸の部分をはんだ付けし直したところ劇的に動作が改善しましたのでここがおそらく原因でした、、、
経過を少し観察しますがこれで問題なく動作するようになったと思います
大変助かりました、本当にありがとうございました！！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
末永くご愛用いただければ幸いです。
